scripts/distribute-devnet-consts.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# copy devnet-consts.json to chain dirs for local use, so we can keep docker
+# build contexts scoped to the chain, rather than the root just to read this file.
+file="./scripts/devnet-consts.json"
+paths="./terra2/tools/
+./wormchain/contracts/tools/"
+
+for dest in $paths; do
```

Comment:
style nit: use an array
```suggestion
paths=(
    ./terra2/tools/
    ./wormchain/contracts/tools/
)

for dest in "${paths[@]}"; do
```

Reply:
good suggestion, fixed